### PR TITLE
Add Prettier & eslint for code style and pattern enforcement

### LINKED ISSUE #608: Formatter
* Config prettier, or similar, ideally on the entire project, but the priority is the development workspace to start
* Should include some automated enforcement mechanism: post commit, on save or similar. 
* PR should include a run of the formatter

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

This adds prettier and eslint with the default svelte configuration to the project.
A workflow is included that verifies that linting passes to ensure that `main` does not get poorly formatted code.

This should have no impact on downstream projects, and the functionality of the project should remain untouched.

### Checklist
- [ ] ~For UI or styling changes, I have added a screenshot or gif showing before & after~
- [ ] ~I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)~

Resolves #608

### CODE REVIEW

**Comment by ItsMeBrianD** [comment]
This is configured correctly; but there are some 400 linting errors to fix. Once most of the currently open PRs are merged; I will merge main into this branch, and then run the formatting and fix the errors to prevent as many merge conflicts as are avoidable.

**Review by ud3sh — APPROVED** [review]

File: packages/preprocess/README.md [hunk 1]
```diff
@@ -33,8 +33,27 @@ flowchart TD
 
 #### Notes:
 
-*1 Evidence will render with the static adapter by default; which means that "runtime" becomes "buildtime".
+\*1 Evidence will render with the static adapter by default; which means that "runtime" becomes "buildtime".
 
-*2 This is handled by the [db-orchestrator package](../db-orchestrator/), not preprocess - but it is an important step.
+<<<<<<< HEAD
```
Comment (Winterhart) [inline packages/preprocess/README.md:38]: here too ✂️

**Review by Winterhart — CHANGES_REQUESTED** [review]
I strongly suggest to start over from another branch as I can't go over the 400 files to make sure the git items are gone (e.g. `<< HEAD`)

**Review by Winterhart — APPROVED** [review]

**Comment by ud3sh** [comment]
@ItsMeBrianD LGM overall.  If I can mention some pet peeves (other than preferring spaces for tabs of course) `printWidth` seems a bit low at 100 (I tend to have my editors full screen).  But overall, this exact value this is far less important than getting it merged IMO.

**Comment by mcrascal** [comment]
LGTM -- only build would be adding the tailwind plugin for class sorting.

**Review by mcrascal — APPROVED** [review]

**Comment by ItsMeBrianD** [comment]
Merging this as is for now; I agree regarding `printWidth`; tailwind class sorting I do think merits a bit of discussion; depends on how we want to style that.

**Pull request merged**